source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx

```diff
@@ -0,0 +1,61 @@
+          <button
+            data-test-sidebar-close-button 
+            className='close' 
+            onClick={toggleSidebar}>
+            <Icon name='close' size='large' />
+          </button>
+        </div>
+
+        <Menu pointing secondary vertical>
+          <Menu.Item 
```

Comment:
have you seen `NavLink`? ;)
https://reacttraining.com/react-router/web/api/NavLink

Reply:
I fixed this, Combined both semantic and NavLink and remove the setActiveMenu